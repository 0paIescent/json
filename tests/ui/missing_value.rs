#[macro_use]
extern crate serde_json;

fn main() {
    json!({ "a" : });
}
