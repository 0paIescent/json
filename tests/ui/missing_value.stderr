error: unexpected end of macro invocation
 --> $DIR/missing_value.rs:5:5
  |
5 |     json!({ "a" : });
  |     ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro outside of the current crate

